usagov/vote-gov-nvrf-app
# Vote-813: Add Download PDF Option

Ticket - [Vote-813](https://cm-jira.usa.gov/browse/VOTE-813)

Purpose - This PR will add the option to download the PDF if a new tab doesn't work for user 

Testing:

1. Fill out form
2. Check that pdf will open in a new tab along with downloading
